Sources/Components/TextField/TextField.swift

```diff
@@ -206,11 +220,17 @@ public class TextField: UIView {
         textField.becomeFirstResponder()
     }
 
+    fileprivate func evaluate(_ regEx: String, with string: String) -> Bool {
+        let regExTest = NSPredicate(format: "SELF MATCHES %@", regEx)
+        return regExTest.evaluate(with: string)
+    }
+
     fileprivate func isValidEmail(_ emailAdress: String) -> Bool {
-        let emailRegEx = "[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,64}"
+        return evaluate("[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,64}", with: emailAdress)
+    }
 
-        let emailTest = NSPredicate(format: "SELF MATCHES %@", emailRegEx)
-        return emailTest.evaluate(with: emailAdress)
+    fileprivate func isValidPassword(_ password: String) -> Bool {
+        return evaluate("{3,64}", with: password)
```

Comment:
Where did you get this from?

Reply:
Yes! I forgot to ask if there’s a specification for password validation in (SPiD). Definitely not 64 char long.

Reply:
It seams to be at least 6 characters long.

Reply:
I don’t think assuming any password validation other than not-empty is a good idea.

Reply:
I think assuming any password validation other than non-empty is a bad idea.

Reply:
ah.. of course!!! Worst case it can lock existing users out. 
But I think this option could be optional/custom validation in future.